webonary-cloud-api/lambda/searchEntries.ts

```diff
@@ -1,3 +1,25 @@
+/**
+ * @api {get} /search/entry/:dictionaryId Search dictionary entries
+ * @apiName SearchDictionaryEntries
+ * @apiDescription Searches the dictionary for entries that match. Returns an array of DictionaryEntryItem's.
+ * (https://github.com/sillsdev/webonary/blob/develop/webonary-cloud-api/lambda/entry.model.ts)
+ * @apiGroup Dictionary
+ * @apiUse DictionaryIdPath
+ * @apiParam {String} text
+ * @apiParam {String} [mainLang] Main language of the dictionary, used for setting the db locale.
+ * @apiParam {String} [lang] Language to search through.
+ * @apiParam {String} [partOfSpeech] Filter results by part of speech.
+ * @apiParam {Number=0,1} [matchPartial] 1 to allow partial matches, and 0 otherwise. Defaults to 0.
+ * @apiParam {Number=0,1} [matchAccents] 1 to match accents, and 0 otherwise. Defaults to 0.
+ * @apiParam {String} [semDomAbbrev] Filter by semantic domain abbreviation.
+ * @apiParam {String} [searchSemDoms] 1 to search by demantic domains, and 0 otherwise. Defaults to 0.
```

Comment:
Should be "semantic domains"

Reply:
Good catch. Fixed in https://github.com/sillsdev/webonary/pull/313/commits/595ef1d96ae6b26ac870d26c0d8531dcf5914d41.